lib/tasks/zomato_api.rake

```diff
@@ -0,0 +1,81 @@
+require 'net/http'
+require 'uri'
+require 'json'
+
+namespace :zomato_api do
+
+  def getResponse uri
+    request = Net::HTTP::Get.new(uri)
+    request["Accept"] = "application/json"
+    request["User-Key"] = "6ad3f6eff60492ea9d4f01379fdbadd6"
+
+    req_options = {
+        use_ssl: uri.scheme == "https",
+    }
+
+    response = Net::HTTP.start(uri.hostname, uri.port, req_options) do |http|
+      http.request(request)
+    end
+
+    return JSON.parse(response.body)
```

Comment:
return is redundant since by default the last returned value in the method is returned.